packages/hash/api/src/db/postgres/aggregation/updateAggregationOperation.ts

```diff
@@ -63,25 +61,18 @@ export const updateAggregationOperation = (
         omittedAggregations: [params],
       });
 
-      sourceEntityVersionIds = new Set([dbSourceEntity.entityVersionId]);
+      updatedDbAggregation.aggregationId = genId();
+      updatedDbAggregation.sourceEntityVersionIds = new Set([
+        dbSourceEntity.entityVersionId,
+      ]);
```

Comment:
I assume we're moving to a timestamp range as part of a follow up?

Reply:
Yes, [the task](https://app.asana.com/0/0/1202032424659149/f) is in the "Next Steps" of the PR description.